site/en/blog/origin-trials/index.md

```diff
@@ -56,16 +56,18 @@ available in Chrome by default. Sites needing extra time to migrate off AppCache
 the [deprecation trial to continue using AppCache](/origintrials/#/view_trial/1776670052997660673) until Chrome 93.
 
 
-## How to register for an origin trial
+## Take part in an origin trial
 
 1. Choose an origin trial from the [list of active trials](https://developers.chrome.com/origintrials/#/trials/active).
-1. Request a token by clicking the **Register** button and filling out the form.
-1. Add the token to your web pages,
-   using one of the following methods:
-   -  As a meta tag in the &lt;head&gt; of each page served:
+1. Request a token (a string of letters and numbers) by clicking the **Register** button and
```

Comment:
Does this definition matter to this audience? I'd guess the developers who will run these instructions know what to expect?

Reply:
You're right — I deleted ' (a string of letters and numbers)'.